web-app/src/pages/Login.vue

```diff
@@ -61,6 +61,8 @@ export default class Login extends Vue {
   private async mounted() {
     if (this.$route.query.code) {
       await api.auth.getCookieToken(this.$route.query.code as string);
+      // const response = await api.organization.getUsers();
+      // await api.organization.getUserPhoto(response[0].id);
```

Comment:
??? Here we should request for user details or better, get this data from token.

Reply:
That was just for tests